project/build.scala

```diff
@@ -244,11 +244,11 @@ object BananaRdfBuild extends Build {
     .settings(
       Seq(
         //scalaJSStage in Test := FastOptStage,
-        jsDependencies += "org.webjars" % "N3.js" % "799fee7697"/ "n3-browser.min.js" commonJSName "N3",
+        jsDependencies += "org.webjars" % "N3.js" % "9a8de1fc6c"/ "n3-browser.min.js" commonJSName "N3",
         skip in packageJSDependencies := false
       ) ++ zcheckJsSettings : _*
     )
-    .dependsOn(rdf_js, rdfTestSuite_js % "test-internal->compile", plantain_js % "test-internal->compile")
+    .dependsOn(rdf_js, plantain_common_js, rdfTestSuite_js % "test-internal->compile", plantain_js % "test-internal->compile")
```

Comment:
You should really dependOn plantain_js as plantain_common_js will not (probably) exist as an artifact at runtime